### PR TITLE
Updating branch for Phase2 in 11_1_X

### PR DESCRIPTION
**Pull request by soureek**
Created a new branch in CMSSW11_1_2 for Phase2 studies. Using the merge topic **emilbols:BTV_CMSSW_11_1_X** from https://github.com/emilbols/RecoBTag-PerformanceMeasurements.git prepared by @emilbols 

The analyzer runs ok with the phase2 option with a few warnings.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,17 +3,16 @@
...
-cd CMSSW_11_0_0_patch1/src
+cmsrel CMSSW_11_1_2 
+cd CMSSW_11_1_2/src
 cmsenv
 
 setenv CMSSW_GIT_REFERENCE /cvmfs/cms.cern.ch/cmssw.git.daily
 git cms-init
 
 git cms-addpkg RecoBTag
-git cms-merge-topic emilbols:BTV_11_0_X
```
Comment (emilbols) [inline README.md:14]: you need here instead:

git cms-merge-topic emilbols:BTV_CMSSW_11_1_X

It replaces the Phase1 DeepJet training with the phase2 one, and ads the phase2 deepcsv training

**Comment by emilbols** [comment]
@Soureek89 Thanks, looks good to me, did you test that it still runs after adding the merge topic? If yes then i think we can merge it.

File: README.md (hunk 1 above)
Comment (soureek) [reply README.md:14]: @emilbols  The branch already had the topic **emilbols:BTV_CMSSW_11_1_X** merged. 
I have updated the instructions in the README (see commit below) and in the twiki (https://twiki.cern.ch/twiki/bin/view/CMS/BTagAnalyzer#Recipe_for_Phase2_11_1_X)

**Comment by soureek** [comment]
@emilbols  Yes, it runs after adding the merge topic.

**Pull request merged**